lib/cocina/generator/schema_array.rb

```diff
@@ -5,7 +5,7 @@ module Generator
     # Class for generating from an openapi array
     class SchemaArray < SchemaBase
       def generate
-        "attribute :#{name.camelize(:lower)}, Types::Strict::Array.of(#{array_of_type}).default([].freeze)"
+        "attribute :#{name.underscore}, Types::Strict::Array.of(#{array_of_type}).default([].freeze)"
```

Comment:
The changes to `lib/cocina/generator/schema_*.rb` are how to translate camelCase OpenAPI attributes to snake_case ones for Ruby collaborators.